src/fs_event_wrap.cc

```diff
@@ -84,8 +84,7 @@ FSEventWrap::FSEventWrap(Environment* env, Local<Object> object)
 }
 
 
-FSEventWrap::~FSEventWrap() {
-}
+FSEventWrap::~FSEventWrap() = default;
```

Comment:
Can you use the `= default` in the class definition instead, like elsewhere?

Reply:
I will handle it late tonight.

Reply:
@addaleax done.